sdk/spring-sdk/src/main/scala/kalix/springsdk/impl/AclDescriptorFactory.scala

```diff
@@ -48,36 +48,30 @@ object AclDescriptorFactory {
 
     aclJavaAnnotation.allow.zipWithIndex.foreach { case (allow, idx) =>
       val principalMatcher = PrincipalMatcher.newBuilder()
-      if (allow.principal != Acl.Principal.UNSPECIFIED) {
-        allow.principal match {
-          case Acl.Principal.ALL =>
-            principalMatcher.setPrincipal(PrincipalMatcher.Principal.ALL)
-          case Acl.Principal.INTERNET =>
-            principalMatcher.setPrincipal(PrincipalMatcher.Principal.INTERNET)
-        }
-        aclBuilder.addAllow(idx, principalMatcher)
-
-      } else {
-        principalMatcher.setService(allow.service())
-        aclBuilder.addAllow(idx, principalMatcher)
+      allow.principal match {
+        case Acl.Principal.ALL =>
+          principalMatcher.setPrincipal(PrincipalMatcher.Principal.ALL)
+        case Acl.Principal.INTERNET =>
+          principalMatcher.setPrincipal(PrincipalMatcher.Principal.INTERNET)
+        case Acl.Principal.UNSPECIFIED =>
+          principalMatcher.setService(allow.service())
```

Comment:
This change in the pattern matching and the one below is just to avoid some compilation warnings.

Reply:
and is easier to read